source/operations/monitoring/metrics-v2.rst

```diff
@@ -72,8 +72,13 @@ The following sections describe the version 2 endpoints and metrics.
 
       .. versionchanged:: RELEASE.2023-08-31T15-31-16Z
 
-      You can scrape :ref:`bucket-level metrics <minio-available-bucket-metrics>` using the following URL endpoint:
+         You can scrape :ref:`bucket-level metrics <minio-available-bucket-metrics>` using the following URL endpoint:
 
+      .. versionchanged:: RELEASE.2025-03-12T17-29-24Z
+
+         v2 metrics have a limit of 100 buckets for performance reasons.
```

Comment:
v2 metrics gives you the first 100 buckets, ordered by something that wasn't quite lexical as I recall. Presumably somebody hitting this would notice that their list of buckets stops at `h` or whatever and figure it out. 

Up to you to decide if that is worth getting into. It's much better for several reasons to move to v3.

Reply:
I think I'll stay out of these weeds for now.